### DISCUSSION TITLE
What is the best way to update NVTs?

### DISCUSSION THREAD

**Discussion by radiusmk** [Secure-Compliance-Solutions-LLC/GVM-Docker#147] [Q&A] [answered]
I have a instalation with master and slaves nodes architeture, so, what is the best way to update NVTs in master and slaves nodes?

Does each slave nodes have own NVT database? If yes, is there any way to update the slave node from the master node? 

How can I see feed status of all slave nodes?

Thanks.
Marco Aurélio

**Comment by pixelsquared** [comment, marked as answer]
For the GVM-Docker right now you can either restart the container or run this to force a NVT update:

```
docker exec -it gvm /sync-all.sh
```
You could setup a cron job or similar to run the above command to schedule the updates.

---

The OpenVAS-Docker image is setup to auto update the NVT feed every 12 hours.

You can see in this script:
https://github.com/Secure-Compliance-Solutions-LLC/OpenVAS-Docker/blob/master/scripts/update-nvts.sh

**Reply by pixelsquared** [reply]
If this helps please mark this as the answer.
